lib/ansible/modules/add_host.py

```diff
@@ -30,26 +30,33 @@
     type: list
     aliases: [ group, groupname ]
 extends_documentation_fragment:
-- action_common_attributes
+  - action_common_attributes
+  - action_common_attributes.conn
+  - action_common_attributes.flow
+  - action_core
 attributes:
     action:
         support: full
-    async:
-        support: none
+    core:
+      details: While parts of this action are implemented in core, other parts are still available as normal plugins and can be partially overridden
+      support: partial
```

Comment:
There are some places where indentation is irregular, but that can easily be fixed later.